src/lib/y2sap/products/read_parameter.rb

```diff
@@ -42,11 +42,13 @@ def read_product_parameter
       log.info("@db #{@db} @product_name #{@product_name} @product_id #{@product_id}")
       eval_product_ay
       setup_installation_enviroment
-      if Yast::Popup.YesNo(_("Installation profile is ready.\n" +
-          "Are there more SAP products to be prepared for installation?"))
+      if Yast::Popup.YesNo(
+        _("Installation profile is ready.\n \
+          Are there more SAP products to be prepared for installation?")
```

Comment:
Missing double quotes, this would include the indentation spaces at the beginning of the line. 

It should be

```ruby
_("Installation profile is ready.\n" \
  "Are there more SAP products to be prepared for installation?")
```

Reply:
Done